utils/modular_model_converter.py

```diff
@@ -709,15 +801,78 @@ def leave_Module(self, original_node: cst.Assign, node):
                 {self.python_module.code_for_node(k): k for k in visiter.imports.values()}
             )
 
+        # Check if any new top-level function should be added to the different files (because it was called in a class in the file).
```

Comment:
```suggestion
        # Check if any new top-level function from the `modular_xxx.py` should be added to the different files (if it is called in a class in the file, then it will be copy pasted from `modular.py` to that file).
```